### DISCUSSION TITLE
How to remove extra domains?

### DISCUSSION THREAD

**Discussion by benbas** [vercel/community#2449] [Help] [answered]
### Summary

How do I remove the extra domains from my project such as https://websitename-benbas.vercel.app/?

**Comment by smaeda-ks** [comment, marked as answer]
Hi @benbas, the "Automatic URL" can't be removed. The deployment must have at least one domain attached, so we always generate one, and this cannot be removed.

**Reply by benbas** [reply]
Thanks for the answer. Are there any plans to change this to be optional in the future?

**Reply by smaeda-ks** [reply]
Not currently. If you're concerned about security, we recommend enabling deployment protection: https://vercel.com/docs/concepts/deployments/deployment-protection (accessing via the "Automatic URL" always treated as preview environment even if it's from a production deployment)

**Reply by benbas** [reply]
ah brilliant, thank you!
